src/net/sf/memoranda/ui/WorkPanel.java

```diff
@@ -225,10 +230,19 @@ else if (pan.equals("TASKS"))
 				tasksB_actionPerformed(null);
 			else if (pan.equals("EVENTS"))
 				eventsB_actionPerformed(null);
-			else if (pan.equals("FILES"))
-				filesB_actionPerformed(null);
+			else if (pan.equals("FILES")) {
+					filesB_actionPerformed(null);
```

Comment:
interesting note: this case can actually trigger on program opening if it's the last panel the user was using. In light of the implemented behavior, that means they could see a folder selection dialog and open folder before memoranda finishes loading.